src/pre-requisites/boardConfigs.ts

```diff
@@ -0,0 +1,413 @@
+};
+
+export const dataBreachCostPreReq: Prerequisites = {
+  groups: [
+    {
+      groupIds: ["cloud"],
+    },
+  ],
+  supportedUseCase:
+    "Shows sensitive data discovery findings from publicly accessible or unencrypted data stores, and use a formula to calculate the potential cost if based on the number of sensitive data records that could be exposed.",
+};
+
+export const dataProtectionPreReq: Prerequisites = {
+  groups: [
+    {
+      groupIds: ["cloud"],
+    },
+  ],
+  supportedUseCase:
+    "Shows DataStore distributions with key metrics on unclassified data, unencrypted data, and non-public data stores with public access.",
+};
+
+export const deviceAnomalyDetectionPreReq: Prerequisites = {
+  groups: [
+    {
+      groupIds: ["device-management"],
+    },
+    {
+      groupIds: ["endpoint-agents"],
+    },
+  ],
+  supportedUseCase:
+    "Shows insights produced by the anomaly detection system (powered by JupiterOne AI). The system scans the *unified_device* entities in your account, and uses advanced machine learning algorithms to detect anomalies in that data set.",
+};
+
+export const deviceManagementPreReq: Prerequisites = {
+  groups: [
+    {
+      groupIds: ["device-management"],
+    },
+    {
+      groupIds: ["endpoint-agents"],
+    },
+  ],
+  supportedUseCase:
+    "Shows Devices missing Host Agents, OS Type and Version Breakdowns, Encryption Status, and Devices with Problems.",
+};
+
+export const gcpComputePreReq: Prerequisites = {
+  groups: [
+    {
+      customGroups: [
+        {
+          category: "Cloud",
+          integrationsNames: ["google_cloud"],
+        },
+      ],
+    },
+  ],
+  supportedUseCase:
+    "Shows important configurations related to GCP Compute Projects and Instances.",
+};
+
+export const gcpIAMPreReq: Prerequisites = {
+  groups: [
+    {
+      customGroups: [
+        {
+          category: "Cloud",
+          integrationsNames: ["google_cloud"],
+        },
+      ],
+    },
+  ],
+  supportedUseCase:
+    "Shows GCP IAM users, roles, service accounts, and important access privilege information.",
+};
+
+export const gdprDataLocationsPreReq: Prerequisites = {
+  groups: [
+    {
+      groupIds: ["cloud"],
+    },
+  ],
+  supportedUseCase: "Shows data stores in the EU, US, and elsewhere.",
+};
+
+export const githubInsightsPreReq: Prerequisites = {
+  groups: [
+    {
+      customGroups: [
+        {
+          category: "Code Management",
+          integrationsNames: ["github"],
+        },
+      ],
+    },
+  ],
+  supportedUseCase:
+    "Shows detailed information on GitHub accounts, reporting on user access, developer activity, collaborator details, insights on pull requests, and issue summaries by repo.",
+};
+
+export const googleWorkshopsPreReq: Prerequisites = {
```

Comment:
I think this is a typo. I was searching for workspace earlier and couldn't find it.
```suggestion
export const googleWorkspacePreReq: Prerequisites = {
```